model/api/src/main/java/org/opennms/oce/model/api/ModelObject.java

```diff
@@ -36,12 +36,36 @@
 
     Set<ModelObject> getChildren();
```

Comment:
This interface assumes there is a single parent, is that an accurate assumption?

Reply:
Yes, always one and only one parent.